examples/matchmaker/src/helpers/bitmap.cairo

```diff
@@ -0,0 +1,490 @@
+// Core imports
+
+use core::integer::BoundedInt;
+use core::debug::PrintTrait;
+
+// Internal imports
+
+use matchmaker::constants;
+
+// Errors
+
+mod errors {
+    const INVALID_INDEX: felt252 = 'Bitmap: invalid index';
+}
+
+#[generate_trait]
+impl Bitmap of BitmapTrait {
```

Comment:
Hmm I wonder if we should contribute this to alexandria?

Reply:
Probably yes, let's do it in another PR to keep flexibility